presto-spark-base/pom.xml

```diff
@@ -302,10 +302,14 @@
                     <ignoredResourcePatterns>
                         <ignoredResourcePattern>parquet.thrift</ignoredResourcePattern>
                         <ignoredResourcePattern>about.html</ignoredResourcePattern>
+                        <ignoredResourcePattern>mozilla/public-suffix-list.txt</ignoredResourcePattern>
                     </ignoredResourcePatterns>
                     <ignoredClassPatterns>
                         <ignoredClassPattern>shaded.parquet.it.unimi.dsi.fastutil.*</ignoredClassPattern>
                         <ignoredClassPattern>module-info</ignoredClassPattern>
+                        <ignoredClassPattern>com.esotericsoftware.kryo.*</ignoredClassPattern>
```

Comment:
Why are we adding these classes as ignored?

Reply:
Yeah, that's right. I tried building on my local. This is needed to fix the build.